src/components/v2/headers/ChartHeader.component.tsx

```diff
@@ -20,7 +20,7 @@ function ChartHeaderComponent({ errorResponseCode }: ChartHeaderComponentType) {
                     to={`${URLS.APP}/${URLS.APP_LIST}/${AppListConstants.AppType.HELM_APPS}`}
                     className="dc__devtron-breadcrumb__item"
                 >
-                    <span className="cb-5 fs-16">Helm Apps </span>
+                    <span className="cb-5 fs-16 pointer">Helm Apps </span>
```

Comment:
Please Use cursor instead of pointer will remove this css someday

Reply:
fixed 👍